android/app/src/androidTest/java/xyz/mobilestack/mobile/DetoxTest.java

```diff
@@ -1,4 +1,4 @@
-package org.celo.mobile;
+package xyz.mobilestack.mobile;
```

Comment:
And removing the intermediary `mobile` folder, so it's shorter overall.
```suggestion
package xyz.mobilestack;
```